examples/medplum-patient-intake-demo/src/bots/core/test-data/intake-form-test-data.ts

```diff
@@ -12,9 +12,208 @@ export const intakePatient: Patient = {
   ],
 };
 
+export const intakeQuestionnaire: Questionnaire = {
```

Comment:
You can actually import the `data/core/patientinakequestionnair` into this test file, to avoid having a to keep them in sync

Reply:
Cool, doing this